layouts/partials/tw/nav.html

```diff
@@ -1,25 +1,33 @@
+<a href="#main" class="sr-only">Skip to main content</a>
+
 <div
-  id="top"
-  class="mx-auto flex max-w-screen-xl flex-wrap items-end justify-between px-3 pt-2 font-sans"
-  data-ga-category="navbar"
+  class="mx-auto flex max-w-screen-xl flex-wrap items-end justify-between bg-beige px-3 pt-2 font-sans"
+  data-ga-category="top-banner"
   data-sf-campaign="701Dn000000YgosIAC"
-  aria-label="main navigation"
+  aria-labelledby="top"
 >
-  <h1
+  <div
+    class="hidden text-xs font-extrabold leading-tight tracking-wide sm:block sm:w-32 lg:w-52 lg:text-sm xl:text-base"
+  >
+    <p>Independent</p>
+    <p>Nonpartisan</p>
+    <p>Nonprofit</p>
+  </div>
+  <div
     class="max-sm:w-44 sm:flex sm:flex-auto sm:items-stretch sm:justify-center"
   >
     <a
+      id="top"
       class="h-max flex-1 sm:mx-auto sm:max-w-xs md:max-w-sm lg:max-w-md"
       href="/"
       data-ga-category="logo"
     >
       <svg class="block h-8 w-full sm:h-14 md:h-16 lg:h-20">
-        <title>Home</title>
-        <desc>Spotlight PA logo</desc>
+        <title>Spotlight PA Home</title>
         {{ partial "helper/svg.html" "banner" }}
       </svg>
     </a>
-  </h1>
+  </div>
```

Comment:
What's the benefit of using a div over h1 here? Using an H1 to define the title in the page header has historically been a best-practice

Reply:
We use H1 for the page title already, so now we have two H1s, one that says Spotlight PA and one that says whatever the page is about.